components/feature/push/src/main/java/mozilla/components/feature/push/AutoPushFeature.kt

```diff
@@ -371,6 +374,9 @@ internal inline fun exceptionHandler(crossinline onError: (PushError) -> Unit) =
         is GeneralError,
         is CryptoError,
         is CommunicationError,
+        is JSONDeserializeError,
+        is RequestError,
```

Comment:
RequestError has historically been a bit noisy, but we can try it out and see if that's still the case.